### PR TITLE
1 6 clean up

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

**Comment by jwkaltz** [comment]
CI is failing because an openlayers commit that cgxp relies upon (indirectly, via submodules) is no longer present in a branch. So it looks like cgxp can no longer be built from scratch:
`error: Server does not allow request for unadvertised object 10e922b46798749f4c6fc17a397189fde1d2e8de
Fetched in submodule path 'demo/static/lib/cgxp/gxp/externals/openlayers', but it did not contain 10e922b46798749f4c6fc17a397189fde1d2e8de. Direct fetching of that commit failed.
`

**Comment by fredj** [comment]
@jwkaltz The openlayers issue should be fixed now, it was an issue with GitHub, see:
https://twitter.com/delawen/status/964152712156925952

**Comment by fredj** [comment]
I've restarted the travis build

**Comment by jwkaltz** [comment]
@fredj thanks but unfortunately we still get the same error.

**Comment by jwkaltz** [comment]
@fredj the OpenLayers commit has still disappeared, but actually I am still not sure what exactly the consequences are:
* A 1.6 project still builds just like before, even after a completely new check-out.
* I created a completely new 1.6 project according to the instructions at https://camptocamp.github.io/c2cgeoportal/1.6/integrator/create_application.html#create-the-new-application, that builds too
* The only thing that triggers this error is when executing the following command in a 1.6 gmf project:
`git submodule update --init --recursive`
Whereas running just `git submodule foreach git submodule update --init` is not a problem.

So bottom line: does it actually matter if `git submodule update --init --recursive` does not work? The only place we do this, that I am aware of, is in the CI integration of demo 1.6.

**Comment by fredj** [comment]
can you run `git submodule sync` to try to fix this git submodule issue?

**Comment by jwkaltz** [comment]
This has no effect in the project :(

**Comment by sbrunner** [comment]
Stalled, closing
